index.html

```diff
@@ -0,0 +1,89 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&family=Roboto:wght@100&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <link rel="stylesheet" href="style.css">
+    <title>Jodrack</title>
+</head>
+<body>
+    <header class="header">
+        <nav class="navbar">
+          <div class="logo"><h1>My Logo</h1></div>
+          <div class="openMenu">
+            <i class="fa fa-bars"></i>
+          </div>
+        </nav>
+    </header>
+
+    <main class="main">
+        <section class="showcase">
+          <div class="showcase__presentation">
+            <h1 class="presentation">I’m Jodrack</h1>
+            <h1 class="sub__presentation">Glad to see you!</h1>
```

Comment:
- [ ] There can be only one `h1` tag on each website page as a title, so let's add `br` tag for breakpoint rather using the `h1` tag on both lines.

Reply:
Great. Let me fix it.